### DISCUSSION TITLE
🙋Nieuwe meter, negatief gebruik in DSMR Reader

### DISCUSSION THREAD

**Discussion by dataegg** [dsmrreader/dsmr-reader#1603] [Questions & answers / How-to's] [answered]
### Description

Hallo,

Ik heb vandaag mijn oude 1 fase slimme meter omgeruild voor een 3 fasen meter. De nieuwe meter stond op 0 en ik zie nu dus een negatief gebruik voor de huidige maand. Hoe kan ik dit het beste oplossen?

mvgr Martin

### DSMR-reader version

4.19

### DSMR-reader platform

Docker (e.g. Xirixiz)

**Comment by dennissiemensma** [comment, marked as answer]
Bedankt voor je vraag. Je kunt (vanaf morgen) het beste #1422 checken/uitvoeren.

**Comment by dataegg** [comment]
Dank voor de snelle reactie, dit heeft inderdaad geholpen.
